src/libcrun/status.c

```diff
@@ -29,6 +29,31 @@
 #include <dirent.h>
 #include <signal.h>
 
+struct pid_stat {
```

Comment:
'{' on the new line.

This is not worth another iteration on its own, but I've another comment left

Reply:
fixed.